services/scanners/results/result_processor.py

```diff
@@ -843,9 +851,7 @@ def process_dns(results, domain_key, uuid, db):
                 elif all(i in ["dkim7", "dkim8"] for i in tags["dkim"][selector]):
                     dkim_statuses.append("pass")
 
-            if len(dkim_statuses) == 0:
-                dkim_status = "fail"
-            elif any(i == "fail" for i in dkim_statuses):
+            if any(i == "fail" for i in dkim_statuses):
                 dkim_status = "fail"
             else:
                 dkim_status = "pass"
```

Comment:
I think this reverts the fix to missing selectors passing, because if there are no selectors dkim_statuses should be empty instead of containing any "fail"s. For now, if we cant assess status for any selectors that should be considered a fail.